src/patternfly/components/Toolbar/toolbar.scss

```diff
@@ -6,7 +6,7 @@ $pf-c-toolbar--inset-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl"
 
 .pf-c-toolbar {
   --pf-c-toolbar--BackgroundColor: var(--pf-global--BackgroundColor--100);
-  --pf-c-toolbar--RowGap: var(--pf-global--spacer--lg);
+  --pf-c-toolbar--RowGap--base: var(--pf-global--spacer--xs);
```

Comment:
This change breaks the `row-gap` of `.pf-c-toolbar`, which looks like it's used when there are multiple `.pf-c-toolbar__content` elements, like in these examples:

* https://patternfly-pr-5398.surge.sh/components/toolbar#selected-filters-on-mobile-filters-collapsed-selected-filters-summary-visible
* https://patternfly-pr-5398.surge.sh/components/toolbar#selected-filters-on-desktop-not-responsive
* https://patternfly-pr-5398.surge.sh/components/toolbar#stacked-on-desktop
* https://patternfly-pr-5398.surge.sh/components/toolbar#stacked-on-mobile-filters-collapsed-expandable-content-expanded
* https://patternfly-pr-5398.surge.sh/components/toolbar#stacked-on-mobile-filters-collapsed-expandable-content-expanded
https://github.com/patternfly/patternfly/blob/25abf8de1fd8ab9579bf67dfc982f411bbb184a5/src/patternfly/components/Toolbar/toolbar.scss#L140